ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py

```diff
@@ -319,16 +323,21 @@ def query_table_names_and_types(
         logic on how to handle table types, e.g., external, foreign,...
         """
 
-        if self.config.serviceConnection.__root__.config.includeTempTables:
-            return [
-                TableNameAndType(name=table_name)
-                for table_name in self.inspector.get_table_names(
-                    schema=schema_name, include_temp_tables="True"
-                )
-                or []
-            ]
-
-        return [
+        regular_tables = [
             TableNameAndType(name=table_name)
-            for table_name in self.inspector.get_table_names(schema=schema_name) or []
+            for table_name in self.inspector.get_table_names(
```

Comment:
@ulixius9 in the payload we send to Openmetadata Apis we need to mark this table as type External

Reply:
@harshach here we are handling the regular tables


[here](https://github.com/open-metadata/OpenMetadata/blob/07f3118c1ef20b98278734377a25a6b210ba26eb/ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py#L335) I'm handling the external tables where I've passed the table type as external as well